lib/client.ts

```diff
@@ -38,6 +39,10 @@ export class Client {
                 break;
         }
         this._serverUrl = `${scheme}://${host}/ws`;
+
+        if (typeof options.token === "string") {
```

Comment:
```suggestion
        if (options.token !== undefined) {
```

Would be good to use a strict check here.